driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -149,6 +163,16 @@ public ConfigBuilder withMinIdleTimeBeforeConnectionTest( long milliSecond )
         }
 
         /**
+         * Enable TLS in all connections with server
+         * @param value
+         */
+        public ConfigBuilder withTLSEnabled( boolean value )
```

Comment:
I like the combination of your two comments: Full TLS (never trust certificates from unknown CAs), Trust-on-first-use (default), and off.